server/testing/src/main/java/org/apache/james/utils/SMTPMessageSender.java

```diff
@@ -113,6 +114,14 @@ public SMTPMessageSender sendMessageWithHeaders(String from, String recipient, S
         return this;
     }
 
+    public SMTPMessageSender sendMessageWithHeaders(String from, List<String> recipients, String message) throws IOException {
+        doHelo();
+        doSetSender(from);
+        recipients.forEach(Throwing.consumer(this::doAddRcpt).sneakyThrow());
```

Comment:
should sendMessageWithHeaders(String, String, String) be rewritten to use sendMessageWithHeaders(String, List<String>, String)?
bonus question: what's the difference between doRCPT and doAddRcpt?

Reply:
`doAddRcpt` put the recipient into brackets, `doRCPT` not.